app/views/post/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="container">
+    <div class="row">
+        <div class="col-3">
+
+        </div>
+        <div class="col-6">
+            <div class="border border-3 rounded p-3 bg-light my-2">
+                <div class="shadown ms-auto d-flex flex-row align-items-center border-bottom m-1">
+                    <p class="text-uppercase text-decoration-none fs-5 link-secondary text-break"> <%= @post.title %> </p>
+                    <p class="text-black-50 mx-2"><%= @post.created_at %></p>
+                </div>
+                <div class="container shadown">
+                    <p class="text-break my-2"><%= @post.body%></p>
+                </div>
+                 <% if user_signed_in? %>
+                    <p class="shadown ms-auto text-end border-bottom py-2">
+                    by <%= @post.user.email %>
+                    </p>
+                <%end%>
+                <% if current_user.id == @post.user_id %>
+                    <div class="d-flex flex-row justify-content-end">
+                        <%= link_to "Edit", edit_user_post_path(current_user.id, @post.id), class:'mx-2 text-decoration-none link-primary' %></li>
+                        <%= link_to "Destroy", user_post_path(current_user.id,@post.id), method: :delete, data: { confirm: "Are you sure?" }, class:'link-danger mx-2 text-decoration-none' %>
+                    </div>
+                <%end%>
+                
+            </div>
+            <div class="border border-3 rounded shadown p-3 my-3">
+                <%= render 'comment/form' %>
+            </div>
+            <div class="border border-3 rounded shadown p-3">
+                <p class="w-100 border-bottom my-3 text-black-50 fs-5">Comments</p>
+                <% if @post.comment.length.zero? %>
+                <p class="text-black-50 text-center">No comments yet</p>
```

Comment:
views should be logic-free. you should not have any logic in the views, specifically `if-else` statements.